test/functional/pdo_sqlsrv/PDO38_FetchBound.phpt

```diff
@@ -40,72 +34,78 @@ function Fetch()
     $stmt2 = PrepareQuery($conn1, "SELECT COUNT(id) FROM [$tableName]");
     $stmt2->execute();
     $num = $stmt2->fetchColumn();
-    echo 'There are ' . $num . " rows in the table.\n";
+    echo "There are $num rows in the table.\n";
     $stmt2->closeCursor();
 
```

Comment:
why closeCursor() here?

Reply:
close the cursor so $stmt2 can be re-executed in line 47 (although not closing the cursor has no problem either, but it's a nice way to ensure the current result set is closed)